OpenUtau/Strings/Strings.axaml

```diff
@@ -188,7 +188,6 @@ Warning: this option removes custom presets.</system:String>
   <system:String x:Key="pianoroll.menu.lyrics">Lyrics</system:String>
   <system:String x:Key="pianoroll.menu.lyrics.dashtoplus">Replace "-" with "+"</system:String>
   <system:String x:Key="pianoroll.menu.lyrics.edit">Edit Lyrics</system:String>
-  <system:String x:Key="pianoroll.menu.lyrics.hanzitopinyin">Hanzi to Pinyin</system:String>
```

Comment:
why remove this?

Reply:
"menu.notes.hanzitopinyin" is used instead of "menu.lyrics.hanzitopinyin"

Reply:
great. thanks